lib/fizzy/parser.cpp

```diff
@@ -77,18 +91,50 @@ struct parser<Global>
 {
     parser_result<Global> operator()(const uint8_t* pos)
     {
-        ValType type;
-        std::tie(type, pos) = parser<ValType>{}(pos);
+        Global result;
+        std::tie(result.is_mutable, pos) = parseGlobalType((pos));
+        std::tie(result.expression, pos) = parser<ConstantExpression>{}(pos);
 
-        if (*pos != 0x00 && *pos != 0x01)
-            throw parser_error{"unexpected byte value " + std::to_string(*pos) +
-                               ", expected 0x00 or 0x01 for global mutability"};
+        return {result, pos};
+    }
+};
 
-        Global result;
-        result.is_mutable = (*pos == 0x01);
-        ++pos;
+template <>
+struct parser<Import>
+{
+    parser_result<Import> operator()(const uint8_t* pos)
+    {
+        std::vector<uint8_t> module;
+        std::tie(module, pos) = parser<std::vector<uint8_t>>{}(pos);
```

Comment:
We should add a parser helper for names. Which must be utf8. @gumb0 can you create a new PR for that?